warsmash.ini

```diff
@@ -0,0 +1,14 @@
+[DataSources]
+Count=6
+Type00=MPQ
+Path00="..\..\war3.mpq"
+Type01=MPQ
+Path01="..\..\War3x.mpq"
+Type02=MPQ
+Path02="..\..\War3xlocal.mpq"
+Type03=MPQ
+Path03="..\..\War3Patch.mpq"
+Type04=MPQ
```

Comment:
Instead of MPQ for the resources, it would probably be better if the github used

```ini
Type04=Folder
Path04="resources/"
```

I figure is critically important that the resources folder does not accidentally include any Blizzard assets for legal reasons -- same reason this entire project cannot include any Blizzard assets -- and seeing we did this correctly is easier to verify with a folder tracked by git than if we include this file as a packaged binary.